codespell_lib/data/dictionary.txt

```diff
@@ -7051,6 +7052,7 @@ correllation->correlation
 correllations->correlations
 correnspond->correspond
 corrensponded->corresponded
+correnspondent->correspondent
```

Comment:
Could add `correnspondents->correspondents,correspondence` ?

Reply:
I've added the first suggestion, the second seems like rather a stretch. I've added the equivalent typo though.